orgs-using-d.dd

```diff
@@ -118,9 +118,9 @@ $(DIVC orgs-using-d center,
     )
     $(DORG Putao, http://www.putao.com, putao.png,
         Science and technology for kids,
-        Large scale use of D for distributed systems and cloud storage.
+        Large scale use of Dlang for distributed systems and cloud storage.
```

Comment:
We are difficult search D in article.
but dlang ok.

I love D used for language name, but dlang  is more practical.